### ISSUE TITLE
P0: One-call tabular AutoML: fit(label) -> predict on a CSV — CRUX-O-01

### ISSUE THREAD

**Issue by noahgift** [paiml/aprender#3371]
Parent: #3370 (CRUX category O — AutoML parity vs AutoGluon 1.6.3) · Tier **P0** · Milestone **0.69.0** · Contract `contracts/crux-O-01-v1.yaml` · demand 5/5 · intake `missing`

## The competitor verb

```python
TabularPredictor(label="class").fit("train.csv"); predictor.predict("test.csv")
```

Source: AutoGluon 1.6.3 checkout (`autogluon/VERSION`), surveyed 2026-09-16.

## The gap (measured in aprender at eb262f8eb)

No predictor-level AutoML entry point exists. crates/aprender-core/src/automl/ is a hyperparameter TUNER (AutoTuner, TPE, GridSearch, RandomSearch, DESearch, TimeBudget) that tunes ONE estimator the caller already chose; nothing takes a labelled table and returns a fitted model. `apr train` is causal-LM pre-training only (crates/apr-cli/src/commands/train.rs:1-5) and `apr finetune --task classify` is text classification.

## Why it ranks where it does

This is the README fold verb of AutoGluon (README.md, the only code block). Every other Category O story is a keyword argument or a method on the object this story creates.

## Acceptance criteria (each must be able to FAIL)

- [ ] **fit on a labelled CSV returns a predictor whose predictions score above the majority-class baseline.** on the iris fixture (crates/aprender-core/src/datasets/iris.csv) accuracy >= 0.90 AND the majority-class baseline is asserted at 0.333 in the same test. Binds to `fit_beats_majority_baseline_on_iris` in `automl/predictor/tests` of `aprender-core` when authored.
- [ ] **the same call works for a regression label without a problem_type argument.** a numeric label column yields a regressor whose R^2 on a held-out split exceeds 0.5 AND the mean-predictor baseline is asserted at ~0.0. Binds to `fit_infers_regression_from_numeric_label` in `automl/predictor/tests` of `aprender-core` when authored.
- [ ] **apr automl fit is reachable from the CLI.** `apr automl fit --label class train.csv --out model.apr` exits 0 and writes a loadable artifact; a missing --label exits 2 with a message naming the flag. Binds to `cli_automl_fit_roundtrip` in `commands/automl_tests` of `apr-cli` when authored.
- [ ] The contract's falsification gates move from LIVE-PENDING to bound test citations, and `pv validate contracts/crux-O-01-v1.yaml` passes with `registry: false`.
- [ ] Mutation-verified: at least one gate above is shown RED under the named mutation before the PR merges.
- [ ] CRUX-SHIP-001: classifier/library code AND `apr` CLI wiring AND one e2e test land in the same PR, or a named blocker (BLOCKER-UPSTREAM-MISSING / BLOCKER-FIXTURE-ABSENT / BLOCKER-SPEC-OPEN) is recorded on the contract.

## Non-goals

Python/pandas API parity. The story is the WORKFLOW (what a user can ask for and what comes back), not the object model.

**Comment by noahgift** [comment]
triage 2026-09-23 (aprender-dd): milestoned to 0.76.0, theme Classical ML + AutoML. It is CRUX category O, AutoML parity vs AutoGluon, which no 0.70–0.75 theme covered. Operator selection 2026-09-23 ("New 0.76 release theme", wording by the cop aprender-cf); moved out of `backlog`.
